tzrec/datasets/dataset.py

```diff
@@ -245,11 +245,28 @@ def get_sampler_cluster(self) -> Optional[Dict[str, Union[int, str]]]:
         if self._sampler:
             return self._sampler._cluster
 
+    def _remove_nullable(self,field_type):
```

Comment:
- add typing
- odps_dataset.py: 88 also should call this function to remove nullable